src/dev/run_check_core_api_changes.ts

```diff
@@ -76,12 +76,16 @@ const apiExtractorConfig = (folder: string): ExtractorConfig => {
 };
 
 const runBuildTypes = async () => {
-  await execa.shell('yarn run build:types');
+  await execa('yarn', ['run', 'build:types']);
 };
 
 const runApiDocumenter = async (folder: string) => {
-  await execa.shell(
-    `api-documenter markdown -i ./build/${folder} -o ./docs/development/core/${folder}`
+  await execa(
```

Comment:
I see you dropped shell from these two commands - just want to confirm that was intentional.

Reply:
Best I could tell these only used shell so they didn't need to specify the arguments as independent values, but doing so makes for better escaping which is a good habit I think and especially important here.